cats/cats-redis/src/main/java/com/netflix/spinnaker/cats/redis/cache/AbstractRedisCache.java

```diff
@@ -234,7 +234,7 @@ protected String allOfTypeId(String type) {
     return String.join(":", prefix, type, "members");
   }
 
-  protected TypeReference getRelationshipsTypeReference() {
+  protected TypeReference<?> getRelationshipsTypeReference() {
```

Comment:
```suggestion
  protected TypeReference<? extends Collection<String>> getRelationshipsTypeReference() {
```

Reply:
Good catch. Any improvement in the types is a good thing. Resolved in 43f237c16f37208c069aceec126271f773eb57b9